README.md

```diff
@@ -42,14 +42,12 @@ Supported targets are:
 
 For go >= 1.16:
```

Comment:
Shouldn't this be changed to Go 1.17 or reworded as it no longer makes sense to mention it?

Reply:
So, the weird bit here is that we can still compile with go 1.14 (as the github action show). I do not really know what we should do/says here. I have not tested with 1.16.

Reply:
Ah, you're right. The new module syntax is compatible to run with older Go versions but developers with them locally will not be able to update/change the module. I guess we should decide on if we want to soft-support older Go or actually raise the version.